spec/unit/state_spec.rb

```diff
@@ -17,12 +17,28 @@ def new_state(options={})
     expect(state.name).to eq(:astate)
   end
 
-  it 'should set the display_name from name' do
-    expect(new_state.display_name).to eq('Astate')
-  end
+  describe '#display_name' do
```

Comment:
These change is necessary because of reason described there https://github.com/aasm/aasm/pull/689/files#r457542825